Tests/js/views/services/plugins/assets/ez-userloadplugin-tests.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-userloadplugin-tests', function (Y) {
+    var tests, registerTest,
+        Assert = Y.Assert;
+
+    tests = new Y.Test.Case({
+        name: "eZ User Load Plugin event tests",
+
+        setUp: function () {
+            this.User = function () {
+                this.set = tests._set;
+                this.load = tests._load;
+            };
+
+            this.userId = '/my/user/42';
+            this.capi = {};
+            this.service = new Y.Base();
+            this.view = new Y.View();
+            this.view.set('myUser', null);
+            this.view.set('loadingError', false);
+            this.view.addTarget(this.service);
+            this.service.set('capi', this.capi);
+
+            this.plugin = new Y.eZ.Plugin.UserLoad({
+                host: this.service,
+            });
+
+            this.plugin.set('userModelConstructor', this.User);
+        },
+
+        _set: function (name, value) {
+            var isCorrectValue = false;
+            Assert.areSame(
+                'id',
+                name,
+                'method set should be called to set the id'
+            );
+            if ( value == tests.userId ){
+                isCorrectValue = true;
+            }
+            Assert.isTrue(isCorrectValue, 'The id should be the user id');
+            this.id = value;
+        },
+
+        _load: function (options, callback) {
+
+        },
+
+        tearDown: function () {
+            this.plugin.destroy();
+            this.view.destroy();
+            this.service.destroy();
+            delete this.plugin;
+            delete this.view;
+            delete this.service;
+        },
+
+        "Should load the related user on `loadUser` event": function () {
+            var that = this;
+
+            this._load = function (options, callback){
+                Assert.areSame(
+                    that.capi,
+                    options.api,
+                    'The REST API client should be passed in the load options'
+                );
+                callback();
+            };
+
+            this.view.fire('loadUser',{
+                userId: this.userId,
+                attributeName: 'myUser'
+            });
+            Assert.areSame(
```

Comment:
this assertion is useless, the correctness of the userId is already checked in `_set`